mastercraft.cmd

```diff
@@ -2037,6 +2103,7 @@ lack.material:
           }
      if "%discipline" = "remed" then
           {
+			   			   if "%order.type" = NULL then return

```

Comment:
Was some bug happening where it would try to order NULL, then there would be no handling for it. Putting this here sorted it.